src/chapters/02/main.tex

```diff
@@ -62,6 +65,10 @@ \section{Theory}\label{sec:theory}
 As customers arrive and leave the system goes between states as shown in
 Figure~\ref{fig:barber-shop-birth-death-no-rates}.
 
+%%% There's a big jump here. Why can't we go directly from state 4 to state 1? We haven't explained.
+%%% We need to explain somewhere that we can go from one state to the other by 1 custoemr arriving or 1 customer leaving.
+%%% In fact, where the rates of change come from is not explained at all!
```

Comment:
I agree **but** this needs us to decide on what level of detail we go to.

Reply:
@geraintpalmer and I have spoken:

- Remove 2.2
- Add a paragraph after application to barber shop: "The state increases when people arrive and this happens at a rate of change of 10. The state decrease when people are served and this happens at a rate of 4 per active server."

Reply:
In previous bullets (with two ways of defining rules) add: "in both cases no two events can occur at the same time".